onnxruntime/core/providers/nnapi/nnapi_builtin/builders/impl/resize_op_builder.cc

```diff
@@ -107,7 +110,6 @@ Status ResizeOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const
   int h_idx = use_nchw ? 2 : 1;
   int w_idx = use_nchw ? 3 : 2;
```

Comment:
If Resize is layout insensitive the use_nchw value won't be applicable. We need to detect the layout.